Tiltfile

```
@@ -95,7 +95,10 @@ COPY --from=tilt-helper /go/kubernetes/client/bin/kubectl /usr/bin/kubectl
 #     }
 # }
 def load_provider_tiltfiles():
-    provider_repos = settings.get("provider_repos", [])
+    provider_repos = settings.get("provider_repos", ["."])
+    if "." not in provider_repos:
+        provider_repos.append(".")
```

Is this still needed with extra_args going in tilt-settings.json now?
```suggestion
    provider_repos = settings.get("provider_repos", [])
```

---

fixed 👍